Extensions/tweaks.js

```diff
@@ -99,6 +99,12 @@ XKit.extensions.tweaks = new Object({
 			text: "User Interface tweaks",
 			type: "separator",
 		},
+		"fix_activity_feed": {
+			text: "Re-slim the Activity feed and highlight notes from mutuals",
```

Comment:
I swear it was highlighting people just as recently as the last time I looked at it? it's not now though...